src/VSCodeExtension/src/generator-qsharp/app/templates/library/library.csproj.txt.v.template

```diff
@@ -0,0 +1,7 @@
+<Project Sdk="Microsoft.Quantum.Sdk/#NUGET_VERSION#">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.1</TargetFramework>
```

Comment:
It may be possible to consolidate this with the other two templates by adding an EPL variable for what the TargetFramework should be, reducing the number of files that need to be maintained.

Reply:
Sounds right, and I considered it. But since the other two sets of templates have separate files for app/lib/test, the pros of keeping them in separate file is that it might be easier to match with them. If at some point in the future something changes in the library project for example, we would update the VS, CLI and yeoman files for that project type only, instead of a common parametrized file for yeoman but separate ones for VS/CLI. What do you think?

Reply:
Makes sense; let's maybe defer, then, any action either way until the rest of the scaffolding is up and working. That would be a pretty easy refactor at the end, in any case.